app/services/food_service.rb

```diff
@@ -0,0 +1,11 @@
+class FoodService
+  def self.search(search_term)
+    query = search_term.gsub(' ','%20') 
+    response = conn.get("/fdc/v1/foods/search?#{query}")
```

Comment:
You have just the search term and not the *name* of the parameter in the url. Instead of `search?query=sweet potatoes` you have `search?sweet potatoes` -- that might be what caused you trouble